client/src/containers/HomePage/HomePage.jsx

```diff
@@ -201,29 +214,28 @@ class HomePage extends Component {
     ) {
       // once you click "next" when the table is open, it will throw an error because the current target (reports) is technically closed
       this.setState({ stepIndex: index + 1 });
-    } else if (index == 7 && [EVENTS.STEP_AFTER].includes(type)) {
-      const { current = {} } = this.mapRef;
-      const { leafletElement: map } = current;
-      map.flyTo(MAP_LOCATIONS.losAngelesCountyCenter, 6 /* ZOOM */, {
-        duration: 0.5,
-      });
-      setTimeout(() => { // Wait for map to pan + zoom to la county then move to next step so target can lock
-        this.setState({ stepIndex: index + (action === ACTIONS.PREV ? -1 : 1) });
-      }, 500);
-    }
+    } // removed the step that automatically zooms onto the map
     else if (
       [EVENTS.STEP_AFTER, EVENTS.TARGET_NOT_FOUND].includes(type)
       && index != 5
     ) {
       // Update state to advance the tour
       this.setState({ stepIndex: index + (action === ACTIONS.PREV ? -1 : 1) });
+    } else if (index == 12) {
+      const context = this.context;
+      if ([EVENTS.TOOLTIP].includes(type)) { // upon mounting of step 12 tooltip, set this context  
+        context.enabled = true;
+        context.stepIndex = 13;
+        context.run = true;
+        console.log(context);
+      }
+    } else if (index == 13) {
```

Comment:
if the user skips over the covid step without clicking on it, make sure tutorial won't run when clicking on covid page